apps/platforms/win/wsl/ubuntu.talon

```diff
@@ -1,11 +1,11 @@
-app: ubuntu
 app: windows_terminal
-and win.title: /Ubuntu/
+win.title: /^WSL:/
```

Comment:
What if instead we require the user.wsl tag as part of the context in the relevant talon files, and leave the tag activation to user-specific for now? We could add comments about your current approach, or include the example commented out.

This keeps it simple, and prevents the issues with the regex out of the box.

Reply:
Sounds good to me. I'll make it so and push the changes some time in the coming days.